kratos/includes/table.h

```diff
@@ -617,21 +617,21 @@ class Table<double, double>
 
     result_type Interpolate(argument_type const& X, argument_type const& X1, result_type const& Y1, argument_type const& X2, result_type const& Y2) const
     {
-        // const double epsilon = 1e-12;
-        return Y1 + (X2 - X1 > 1.0e-12) * (Y2 - Y1)*(X - X1)/(X2 - X1);
+        constexpr double epsilon = 1.0e-12;
```

Comment:
I think that what @pooyan-dadvand meant was to define the `epsilon` as a class `constexpr` so you ensure that a unique value is used.